content/id/docs/tasks/administer-cluster/configure-upgrade-etcd.md

```diff
@@ -0,0 +1,234 @@
+
+Mencadangkan klaster etcd dapat dilakukan dengan dua cara: _snapshot_ etcd bawaan dan snapshot volume.
+
+### Snapshot bawaan
+
+etcd mendukung _snapshot_ bawaan, jadi mencadangkan klaster etcd lebih mudah. _snapshot_ dapat diambil dari anggota langsung dengan command `etcdctl snapshot save` atau dengan menyalin `member/snap/db` file dari etcd [direktori data](https://github.com/coreos/etcd/blob/master/Documentation/op-guide/configuration.md#--data-dir) yang saat ini tidak digunakan oleh proses etcd. Mengambil snapshot biasanya tidak akan memengaruhi kinerja anggota.
+
+Di bawah ini adalah contoh untuk mengambil snapshot dari _keyspace_ yang dilayani oleh `$ENDPOINT` ke file `snapshotdb`:
+
+```sh
+ETCDCTL_API=3 etcdctl --endpoints $ENDPOINT snapshot save snapshotdb
+# exit 0
+
+# verify the snapshot
+ETCDCTL_API=3 etcdctl --write-out=table snapshot status snapshotdb
++----------+----------+------------+------------+
+|   HASH   | REVISION | TOTAL KEYS | TOTAL SIZE |
++----------+----------+------------+------------+
+| fe01cf57 |       10 |          7 | 2.1 MB     |
++----------+----------+------------+------------+
+```
+
+### Snapshot volume
+
+Jika etcd berjalan pada volume penyimpanan yang mendukung cadangan, seperti Amazon Elastic Block Store, buat cadangan data etcd dengan mengambil snapshot dari volume penyimpanan.
+
+## Scaling up klaster etcd
+
+Penskalaan klaster etcd meningkatkan ketersediaan dengan menukar kinerja. Penskalaan tidak meningkatkan kinerja atau kemampuan klaster. Aturan umum adalah untuk tidak _scale up_ atau _scale down_ klaster etcd. Jangan mengkonfigurasi grup penskalaan otomatis untuk klaster etcd. Sangat disarankan untuk selalu menjalankan klaster etcd five-member statis untuk klaster Kubernetes produksi pada skala yang didukung secara resmi.
+
+Penskalaan yang wajar adalah untuk meningkatkan klaster three-member menjadi five-member, ketika dibutuhkan lebih banyak reliabilitas. Lihat [Dokumentasi Rekonfigurasi etcd](https://github.com/coreos/etcd/blob/master/Documentation/op-guide/runtime-configuration.md#remove-a-member) untuk informasi tentang cara menambahkan anggota ke klaster yang ada.
+
+## Memulihkan klaster etcd
+
+etcd mendukung pemulihan dari snapshot yang diambil dari proses etcd dari versi [major.minor](http://semver.org/). Memulihkan versi dari versi patch lain dari etcd juga didukung. Operasi pemulihan digunakan untuk memulihkan data klaster yang gagal.
+
+Sebelum memulai operasi pemulihan, file snapshot harus ada. Ini bisa berupa file snapshot dari operasi cadangan sebelumnya, atau dari yang sisa [direktori data](https://github.com/coreos/etcd/blob/master/Documentation/op-guide/configuration.md#--data-dir). Untuk informasi dan contoh lebih lanjut tentang memulihkan klaster dari file snapshot, lihat [dokumentasi pemulihan bencana etcd](https://github.com/coreos/etcd/blob/master/Documentation/op-guide/recovery.md#restoring-a-cluster).
```

Comment:
```suggestion
Sebelum memulai operasi pemulihan, berkas _snapshot_ harus ada. Ini bisa berupa berkas _snapshot_ dari operasi pencadangan sebelumnya, atau dari [direktori data] yang tersisa (https://github.com/coreos/etcd/blob/master/Documentation/op-guide/configuration.md#--data-dir). Untuk informasi dan contoh lebih lanjut tentang memulihkan klaster dari berkas _snapshot_, lihat [dokumentasi pemulihan bencana etcd](https://github.com/coreos/etcd/blob/master/Documentation/op-guide/recovery.md#restoring-a-cluster).
```

Reply:
Hello, @ariscahyadi Thank you for reviewing my PR.
I prefer this change `atau sisa dari [direktori data](https://github.com/coreos/etcd/blob/master/Documentation/op-guide/configuration.md#--data-dir).`
Because it's not removing the markdown link.